extensions/amp-story/1.0/amp-story-store-service.js

```diff
@@ -68,6 +68,16 @@ export let InteractiveComponentDef;
  */
 export let InteractiveReactData;
 
+/**
+ * No effect on runtime. Merely an annotation for the compiler to shorten the
+ * property values of large, common enums during production.
+ * See babel-plugin-mangle-object-values.
+ * @param {T} obj
+ * @return {T}
+ * @template T
+ */
+const mangleObjectValues = (obj) => obj;
```

Comment:
Can we have this in story/utils.js so it's easier to reuse?

Reply:
Actually, let's move this into `#core/types/enums`. There are a few places in core where we're using `@enum {number}` which would be easier to debug if we make them manageable strings.